### PR TITLE
Network: added a basic web UI to PPPGW

### PR DESCRIPTION
**Pull request by tridge**
fixes scripting in AP_Periph with:
 - fixed fw versioning
 - networking access in scripts
 - uart access in scripts
 - param access and table creation in scripts

image

### CODE REVIEW

**Review by magicrub — APPROVED** [review]
BOOHYA! Me like

File: libraries/AP_Scripting/generator/description/bindings.desc [hunk 1]
```diff
@@ -888,3 +887,11 @@ singleton AP_RTC depends AP_RTC_ENABLED
 singleton AP_RTC rename rtc
 singleton AP_RTC method clock_s_to_date_fields boolean uint32_t'skip_check uint16_t'Null uint8_t'Null uint8_t'Null uint8_t'Null uint8_t'Null uint8_t'Null uint8_t'Null
 singleton AP_RTC method date_fields_to_clock_s uint32_t uint16_t'skip_check int8_t'skip_check uint8_t'skip_check uint8_t'skip_check uint8_t'skip_check uint8_t'skip_check
+
+include AP_Networking/AP_Networking.h depends AP_NETWORKING_ENABLED
```
Comment (magicrub) [inline libraries/AP_Scripting/generator/description/bindings.desc:891]: CI is complaining about not knowing AP_NETWORKING_ENABLED

File: libraries/AP_Scripting/generator/description/bindings.desc [hunk 2]
```diff
@@ -888,3 +887,11 @@ singleton AP_RTC depends AP_RTC_ENABLED
...
+singleton AP_Networking depends AP_NETWORKING_ENABLED
+singleton AP_Networking rename networking
```
Comment (magicrub) [inline libraries/AP_Scripting/generator/description/bindings.desc:893]: Needs another line to include the config
`include AP_Networking/AP_Networking_config.h`

File: libraries/AP_Scripting/generator/description/bindings.desc (hunk 1 above)
Comment (tridge) [reply libraries/AP_Scripting/generator/description/bindings.desc:891]: fixed

File: libraries/AP_Scripting/generator/description/bindings.desc (hunk 2 above)
Comment (tridge) [reply libraries/AP_Scripting/generator/description/bindings.desc:894]: fixed

**Review by magicrub — APPROVED** [review]

**Pull request merged**